app/webserver/webserver.go

```diff
@@ -263,11 +261,12 @@ func wsReader(conn *websocket.Conn) {
 			setPasswordMessage := SetPasswordMessage{}
 			json.Unmarshal([]byte(p), &setPasswordMessage)
 			log.Infoln("[axolotl] set password")
-			if settings.SettingsModel.EncryptDatabase {
-				if !store.DS.DecryptDb(setPasswordMessage.CurrentPw) {
-					// setError(i18n.tr("Incorrect old passphrase!"))
-				}
-			}
+			// TODO: proof current password is correct
```

Comment:
What is required for this code/functionality to be brought back?
It is not clear to me at the moment.

Reply:
This is actually working but what's missing is the error handling and showing in the UI that the old password was wrong.

Reply:
This needs a fast follow up before the next release